README.md

```diff
@@ -1,2 +1,20 @@
-# Journal App
-By Aaron, James & Ravil
\ No newline at end of file
+![I need a <br/>](https://i.imgur.com/Up0POfW.png)
+
+## By Aaron, James & Ravil
+
+## Initallation Instructions
+Our app is to be run on your local machine so do the following:
+1. Clone this repo (from main) in your preferred method
+2. Navigate to the root directory of this repo
+3. In your terminal run "bash _scripts/startDev.sh" (give it a few mins to install dependencies)
+4. Once ready navigate to localhost:8080 to view the clientside and localhost:3000 to view the API.
+5. When restarting or stopping, we advice you teardown using "bash _scripts/teardown.sh"
+
+## What to play around with:
+Try the following:
+- Play around with adding your own thoughts about code or anything.
+- We have 3 pre-made posts, to navigate to them use their ID which will be 1, 2 or 3 as so: "localhost:8080/#2" (make sure you include the #)
+- Try navigating to a post ID that does not exist, try: "localhost:8080/#10"
+
```

Comment:
Some nice suggestions here for the user.  We tried them out and everything worked as expected!